web/api/backend.go

```diff
@@ -0,0 +1,127 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"time"
+	jwtmiddleware "github.com/auth0/go-jwt-middleware"
+	"github.com/dgrijalva/jwt-go"
+)
+
+// Credentials object read body from the request body
+type Credentials struct {
+	RequestType string
+	Args        map[string]interface{}
+	UseCache    string
+	Body        map[string]interface{}
+}
```

Comment:
Would we need a field for the optional token storing the credential?

Reply:
Resolved